tests/rule_based_profiler/conftest.py

```diff
@@ -1067,3 +1071,32 @@ def profiler_with_placeholder_args(
         **profiler_config_dict, data_context=empty_data_context
     )
     return profiler
+
+
+@pytest.fixture
+def usage_stats_profiler_config() -> RuleBasedProfilerConfig:
```

Comment:
@cdkini This seems close to our other profiler configuration fixture(s) -- any chance we can use that (with a potential modification)?  Even if we have to update a lot of existing tests a little bit, I feel that it could be worth it.  Thanks!